VisualizationBase/styles/item/ArrowOverlay/default

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style>
```

Comment:
Instead of specifying all these properties, always specify a prototype and only specify the properties that differ. At a minimum use ../Item/default